fog/sample-paykit/src/cached_tx_data/mod.rs

```diff
@@ -32,6 +35,10 @@ pub use memo_handler::{MemoHandler, MemoHandlerError};
 
 const MAX_INPUTS: usize = mc_transaction_core::constants::MAX_INPUTS as usize;
 
+/// Highest subaddress index we support.
```

Comment:
```suggestion
/// Highest subaddress index we support.
/// If TxOut's are found which belong to an unsupported subindex, this will be detected during matching,
/// and a "SubaddressNotFound" error will be returned, and the client will not spend this TxOut, and the
/// balance of this account will not reflect this subaddress.
```